ci/test/main.go

```diff
@@ -28,6 +28,8 @@ func main() {
 		os.Exit(1)
 	}
 
+	_ = os.Setenv("ORIGIN", *origin)
```

Comment:
The scripts were requesting this as an environment variable, but nobody was setting it.

Origin comes from a command line parameter, and it's not an environment variable.
So we explicitly convert it for the rest of the script to use it.